### PR TITLE
Civilian Bounties now offer a choice of three different bounties to pick from.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
Credits goes to @spessbro for doing most of the backend while I was trapped with no internet for about a week.

## About The Pull Request
The civilian bounty console now offers a choice of 3 different bounties when getting a new civilian bounty choice.
The first two of these options will be different bounty types from within your jobs pool, while the third will always be a generic bounty (AKA from the pool that assistants and various misc. jobs choose from). 
This gives the player a bit more choice than they had before in regards to doing a task related to their job within the existing bounty system until we add a more concrete method for some of the more poorly aligned bounty pools to be brought in-line with existing gameplay.
Also it offers a way to make economy impact more interactive with the round.

image

## Why It's Good For The Game

Bounties need a nice clean sweep to keep them in-line with existing gameplay loops, but the few bounties we have that are in-line with how the game is played can still completely fuck you over when your assigned to get something specific enough or random enough that you'll never see it in regular gameplay (Like getting 7 consecutive "Lizard Tail" bounties).

This offers a modicum of choice, as well as allows for an acceptable alternative for jobs whose specific outputs are not going to be met by the majority of players.

## Changelog
:cl:ArcaneMusic and SpessBro
balance: Civilian bounties now offer a choice of 3 applicable bounties instead of a single random one, with 2 being job specific bounties and one being a guaranteed generic bounty.
/:cl:

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
I don't like the UI, this comes to mind:

https://twitter.com/steveschoger/status/997125312411570176

> Don't be afraid to "think outside the database" — your UI doesn't need to map one-to-one with your data's fields and values.
Here are a few ideas you can use to present "field: value" data in a more interesting way:

image

**Comment by Tlaltecuhtli** [comment]
this is for bounty computer not to buy house

**Comment by Jared-Fogle** [comment]
user interfaces are interfaces for users

**Comment by Owai-Seek** [comment]
If it's a choice between several bounties, and not a thing where you can do -all- of them at once, it may be best to highlight that.

IE: 
```
Option 1
Option 2
Option 3
```
or
```
Choice 1
Choice 2
Choice 3
```

As a sidenote, what does this look like on an ID when you examine it closely?

**Comment by ArcaneMusic** [comment]
Made some changes to the UI style now that tgui is working again on my machine:
image

**Comment by ArcaneMusic** [comment]
More additional changes with margins and flex spacing
image

File: tgui/.yarnrc.yml [hunk 1]
```diff
@@ -1 +1 @@
-yarnPath: .yarn/releases/yarn-2.3.3.cjs
+yarnPath: .yarn/releases/yarn-2.4.0.cjs
```
Comment (stylemistake) [inline tgui/.yarnrc.yml]: Revert changes to this file and `.yarn/releases/yarn-2.x.x.cjs`, we don't want this to conflict with CBT for no reason, and upgrading yarn is not necessary for it to function.

Comment (ArcaneMusic) [reply tgui/.yarnrc.yml]: Yup, handled that yesterday.

**Pull request merged**